### ISSUE TITLE
Style/VariableName cop ignores optional parameters

### ISSUE THREAD

**Issue by JaniJegoroff** [bbatsov/rubocop#4555]
## Expected behavior
```
Inspecting 1 file
C

Offenses:

test.rb:1:10: C: Style/VariableName: Use snake_case for variable names.
def test(aFoo = nil)
         ^^^^

1 file inspected, 1 offense detected
```

## Actual behavior
```
Inspecting 1 file
.

1 file inspected, no offenses detected
```

## Steps to reproduce the problem

Given test1.rb - Offense is reported correctly with normal parameter.
```
def test(aFoo)
  puts aFoo
end

$ rubocop -D test1.rb

test.rb:1:10: C: Style/VariableName: Use snake_case for variable names.
def test(aFoo)
         ^^^^

1 file inspected, 1 offense detected
```

Given test2.rb - Offense is ignored when having optional parameter.
```
def test(aFoo = nil)
  puts aFoo
end

$ rubocop -D test2.rb

Inspecting 1 file
.

1 file inspected, no offenses detected
```

## RuboCop version
```
$ rubocop -V
0.49.1 (using Parser 2.4.0.0, running on ruby 2.2.5 x86_64-darwin15)
```
